webui/react/src/components/TimeAgo.tsx

```diff
@@ -25,7 +23,6 @@ export const TimeAgoCase = {
 export type TimeAgoCase = ValueOf<typeof TimeAgoCase>;
 
 interface Props {
-  className?: string;
```

Comment:
https://github.com/determined-ai/determined/blob/f58038533742ea20ba9df1c7ea5bfc097592015d/webui/react/src/components/TimeAgo.test.tsx#L50

please fix the test file